### ISSUE TITLE
Would you want to add this to the video.js plugins list?

### ISSUE THREAD

**Issue by heff** [themotleyfool/videojs-ooyala#2]
https://github.com/videojs/video.js/wiki/Plugins

**Comment by aendrew** [comment]
@adamghill If you need some help updating or maintaining this, my team (begrudgingly) uses Ooyala a lot, and your library is the best and easiest way of getting it to work with VideoJS I've run across. Let me know if interested!

**Comment by adamghill** [comment]
@aendrew Yes, definitely! We are using Ooyala less and less now, so we haven't been keeping this plugin up to date. I can give you access to push to this repo, or you can fork and I can update the readme to mention your repo as actually being updated. Just let me know, and thanks for the offer!

**Comment by aendrew** [comment]
@adamghill Sounds good! Will query my team on Monday and discuss which is the better option.

**Comment by heff** [comment]
Glad to hear this project is gonna keep going! Definitely add it to the plugin wiki when you figure out how you're gonna handle it.
